README.md

```diff
@@ -19,18 +19,15 @@
 
 <hr />
 
-Lance makes machine learning workflows with ML data easy (images, videos, point clouds, audio, and of course tabular data), by allowing Developers, Analysts and Operations to:
+Lance is a columnar data format that is easy and fast to version, query and train on. It’s designed to be used with images, videos, 3D point clouds, audio and of course tabular data. The key features of Lance include:
 
-* Use SQL to greatly simplify common operations on ML data, such as similarity search for data discovery, model inference and computing evaluation metrics.
+* **High-performance random access:** 100x faster than Parquet.
 
-* Search for nearest neighbors in under 1 millisecond.
+* **Embedded vector search:** find nearest neighbors in under 1 millisecond and combine OLAP-queries with vector search.
```

Comment:
Same here. Ppl will think we're confusing embedded vs embedding

Reply:
Ack, modified